android/app/build.gradle

```diff
@@ -116,11 +116,11 @@ dependencies {
   implementation 'androidx.multidex:multidex:2.0.0'
 
   // Our dependencies
-  implementation 'androidx.appcompat:appcompat:1.0.0'
+  implementation 'androidx.appcompat:appcompat:1.1.0'
 
   // Our dependencies from ExpoView
   // DON'T ADD ANYTHING HERE THAT ISN'T IN EXPOVIEW. ONLY COPY THINGS FROM EXPOVIEW TO HERE.
-  implementation 'androidx.appcompat:appcompat:1.0.0'
+  implementation 'androidx.appcompat:appcompat:1.1.0'
```

Comment:
Aren't these the same?

![Zrzut ekranu 2019-11-12 o 10 35 49](https://user-images.githubusercontent.com/1151041/68659777-32fd7400-0538-11ea-90dc-46df99df081b.png)